Added note about Z-index incompatibility (#1288)
vaadin/docs: articles/tools/mpr/configuration/limitations.asciidoc

= Limitations of MPR

// Disable 
pass:[<!-- vale Vaadin.Headings = NO -->]

> Using MPR in your project to port a legacy application to Vaadin Flow has some known limitations. This is the current list of limitations - keep in mind that it can change over time as new features are implemented.

Reviewer: ```suggestion
Using MPR in your project to port a legacy application to Vaadin Flow has some known limitations.
This is the current list of limitations.
Keep in mind that it may change over time as new features are implemented.
```

> == It's possible to add a legacy component to a Flow layout, but not a Flow component in a legacy layout

Reviewer: ```suggestion
== It is possible to add a legacy component to a Flow layout, but not a Flow component to a legacy layout
```

> Since Vaadin Board 1.1.0 requires Polymer 2 and Vaadin 14 requires Polymer 3, Vaadin Board is not compatible with Vaadin 14 and cannot be used as a legacy component. It works if Vaadin 14 runs in compatibility mode.

Reviewer: ```suggestion
Since Vaadin Board 1.1.0 requires Polymer 2 and Vaadin 14 requires Polymer 3, Vaadin Board is not compatible with Vaadin 14 and cannot be used as a legacy component.
It works if Vaadin 14 is running in compatibility mode.
```

== Z-indexes of the modern Vaadin Design System are not aligned with legacy

> Components using overlays / popups in Vaadin 7 and Vaadin 8 are organized in different z-indexes than similar components in the modern Vaadin Design System. Thus it can be possible that some combinations of them does not work. For example using such Vaadin 7 / 8 components with LegacyWrapper in Dialog may work un-expectly.

Reviewer: ```suggestion
Components using overlays / popups in Vaadin 7 and Vaadin 8 are organized in different z-indexes from similar components in the modern Vaadin Design System.
Thus it is possible that some combinations of them do not work.
For example, using such Vaadin 7 / 8 components with LegacyWrapper in Dialog may not work as expected.
```
